R/f.R

```diff
@@ -6,8 +6,8 @@
 #' @export
 #' @examples
 #' new_f(quote(a))
-#' new_f(quote(a), quote(b))
-new_f <- function(rhs, lhs = NULL, env = parent.frame()) {
+#' new_f(quote(a), lhs = quote(b))
+new_f <- function(rhs, env = caller_env(), lhs = NULL) {
```

Comment:
ok. Maybe have `new_quote(rhs, env)` in addition to `new_f(rhs, lhs, env)`? And then the latter can have the less confusing argument order `new_f(lhs, rhs, env)`